crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -445,6 +451,28 @@ def delete_pacemaker_resources(node_name)
         end
       end
 
+      # Evacuate all routers away from the specified network node to other
+      # available network nodes. The evacuation procedure it started on the
+      # specified controller node
+      def evacuate_network_node(controller, network_node, delete_namespaces = false)
```

Comment:
So the `controller` node is the target node for the routers, right? Wouldn't it be better name?
(Feel free to discard this comment, if network experts think it is better this way, than it likely is)